Display ratings for episodes

K3UL opened the pull request:
If the setting is set to display TV shows ratings, the ratings for episodes are also individually shown.

I didn't want to make the settings too cluttered so I included it in the "global" tv show ratings option, as I also think if you want one, you want the other too. I could make a separate setting if necessary.

K3UL committed:
Display ratings for episodes

If the setting is set to display TV shows ratings, the ratings for episodes are also individually shown.

jeroenpardon commented:
Thanks :) Makes sense to include it. However there is one problem with this PR. The setting does not conform to the "Only show user ratings" skin setting. If said option is on, ratings pulled from thetvdb, imdb, etc. should not show and only ratings set by the user are shown.

Having this as a global setting seems fine to me.

K3UL committed:
Fixed OSD in case of missing rating

Fixed OSD ; now when rating is enabled for episode but there's not rating to show, the bullet is there to separate the label and video resolution.

K3UL committed:
Setting for tv episodes ratings

Ratings for individual episodes, with global option and use of user ratings if wanted

K3UL commented:
I made it a global setting ; it is independant from the "show tv shows ratings" as it would have weird behavior if selected but disabled. Makes more sense to have it just always available.
I also fixed it so it can use user rating if the setting is enabled.

Finally, I added the english string and the french translation (I'm french).

jeroenpardon reviewed (changes requested):
Thanks, but there's quite a few changes / reverts here that aren't (or shouldn't be) related to the tv / episode ratings feature. Such as using include name instead of include content. Be aware the master branch is Krypton. Please pull from the master branch and apply the changes there.

New strings only need to be added to the english language file, the other languages are dealt with through Transifex.
